### PR TITLE
Fixed few test misses on OSX

### PR DESCRIPTION
**Pull request by AshwiniChalla**
Path seperator was not tested on OSX. Added that.

### CODE REVIEW

File: Tests/L0/ANT/_suite.ts [hunk 1]
```diff
@@ -307,76 +307,75 @@ describe('ANT Suite', function() {
...
+
+        tr.run()
+            .then(() => {
+                assert(tr.stdout.search(/##vso\[codecoverage.enable buildfile=\/build\/build.xml;classfilesdirectories=class1;summaryfile=coverage.xml;reportdirectory=\\build\\CCReport43F6D5EF;ccreporttask=CodeCoverage_9064e1d0;reportbuildfile=\\build\\CCReportBuildA4D283EG.xml;buildtool=Ant;codecoveragetool=Cobertura;\]/) >= 0 || tr.stdout.search(/##vso\[codecoverage.enable buildfile=\/build\/build.xml;classfilesdirectories=class1;summaryfile=coverage.xml;reportdirectory=\/build\/CCReport43F6D5EF;ccreporttask=CodeCoverage_9064e1d0;reportbuildfile=\/build\/CCReportBuildA4D283EG.xml;buildtool=Ant;codecoveragetool=Cobertura;\]/) >= 0, 'should have called enable code coverage.');
+                assert(tr.stdout.search(/##vso\[codecoverage.publish codecoveragetool=Cobertura;summaryfile=\\build\\CCReport43F6D5EF\\coverage.xml;reportdirectory=\\build\\CCReport43F6D5EF;\]/) >= 0 ||
+                    tr.stdout.search(/##vso\[codecoverage.publish codecoveragetool=Cobertura;summaryfile=\/build\/CCReport43F6D5EF\/coverage.xml;reportdirectory=\/build\/CCReport43F6D5EF;\]/) >= 0, 'should have called publish code coverage.');
```
Comment (nigurr) [inline Tests/L0/ANT/_suite.ts]: indention/not asserting?

File: Tests/L0/ANT/_suite.ts [hunk 2]
```diff
@@ -307,76 +307,75 @@ describe('ANT Suite', function() {
...
-    //         });
-    // })
+        tr.run()
+            .then(() => {
+                assert(tr.stdout.search(/##vso\[codecoverage.enable buildfile=\/build\/build.xml;classfilesdirectories=class1;summaryfile=coverage.xml;reportdirectory=\\build\\CCReport43F6D5EF;ccreporttask=CodeCoverage_9064e1d0;reportbuildfile=\\build\\CCReportBuildA4D283EG.xml;buildtool=Ant;codecoveragetool=JaCoCo;\]/) >= 0 || tr.stdout.search(/##vso\[codecoverage.enable buildfile=\/build\/build.xml;classfilesdirectories=class1;summaryfile=coverage.xml;reportdirectory=\/build\/CCReport43F6D5EF;ccreporttask=CodeCoverage_9064e1d0;reportbuildfile=\/build\/CCReportBuildA4D283EG.xml;buildtool=Ant;codecoveragetool=JaCoCo;\]/) >= 0, 'should have called enable code coverage.');
+                assert(tr.stdout.search(/##vso\[codecoverage.publish codecoveragetool=JaCoCo;summaryfile=\\build\\CCReport43F6D5EF\\coverage.xml;reportdirectory=\\build\\CCReport43F6D5EF;\]/) >= 0 ||
+                    tr.stdout.search(/##vso\[codecoverage.publish codecoveragetool=JaCoCo;summaryfile=\/build\/CCReport43F6D5EF\/coverage.xml;reportdirectory=\/build\/CCReport43F6D5EF;\]/) >= 0, 'should have called publish code coverage.');
```
Comment (nigurr) [inline Tests/L0/ANT/_suite.ts]: same here

**Comment by nigurr** [comment]
is there any RB running against it? If not, we should setup RB for coreClr agent

File: Tests/L0/ANT/_suite.ts (hunk 1 above)
Comment (AshwiniChalla) [inline Tests/L0/ANT/_suite.ts]: already indented

File: Tests/L0/ANT/_suite.ts (hunk 2 above)
Comment (AshwiniChalla) [inline Tests/L0/ANT/_suite.ts]: same here

**Comment by AshwiniChalla** [comment]
We have a user story for RB for coreclr agent

**Pull request merged**